apps/ejabberd/src/mod_pubsub.erl

```diff
@@ -235,6 +236,10 @@ stop(Host) ->
     ok = supervisor:terminate_child(ejabberd_sup, Proc),
     ok = supervisor:delete_child(ejabberd_sup, Proc).
 
+-spec default_host() -> string().
+default_host() ->
+    "pubsub.@HOST@".
```

Comment:
I see that before the change it was a binary. Why is it changed to plain string now?